src/components/ManageUsers/ResetPassword/index.js

```diff
@@ -11,7 +11,7 @@ import PasswordResetForm from "./PasswordResetForm";
 
 const { Title } = Typography;
 
-const ResetPassword = ({ queryParams }) => {
+const ResetPassword = ({ queryParams = "test" }) => {
```

Comment:
why you are passing 'test' here? Any specific reason?

Reply:
actually in the line 17 , we have used queryParams. Basically we have to pass a code when from different component when we call this ResetPassword/index.js component. I have set default value of queryParams as "test". otherwise it will throw undefined error.